<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personaldoc</class>
 <widget class="QWidget" name="personaldoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>791</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblAvatar">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUploadAvatar">
        <property name="text">
         <string>上传头像</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>昵称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNickname"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>性别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxGender"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>211</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>已绑定邮箱：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QCalendarWidget" name="startDateEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>296</width>
       <height>236</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="setPomodoros">
        <property name="text">
         <string>完成的番茄钟个数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setDuration">
        <property name="text">
         <string>总时长</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setPauses">
        <property name="text">
         <string>暂停次数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>生成柱状图</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCalendarWidget" name="endDateEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>296</width>
       <height>236</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>成就系统</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>95</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnPersonalInfo">
      <property name="text">
       <string>个人信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHistory">
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAchievements">
      <property name="text">
       <string>成就</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
